core/controllers/admin.py

```diff
@@ -129,6 +136,8 @@ def get(self):
             'continuous_computations_data': continuous_computations_data,
             'demo_explorations': demo_explorations,
             'demo_exploration_ids': demo_exploration_ids,
+            'demo_collections': demo_collections,
```

Comment:
oh, no, sorry. I meant alphabetize keys in this dict, so the 'demo_collections' key goes above 'demo_explorations'.